DQMOffline/Muon/src/GEMEfficiencyAnalyzer.cc

```diff
@@ -107,7 +109,8 @@ void GEMEfficiencyAnalyzer::bookHistograms(DQMStore::IBooker& ibooker,
                                            edm::Run const& run,
                                            edm::EventSetup const& isetup) {
   edm::ESHandle<GEMGeometry> gem;
-  isetup.get<MuonGeometryRecord>().get(gem);
+  gem = isetup.getHandle(gemToken_);
```

Comment:
The failures are coming from here. Since the transition ID stored for geomToken_ is Event and here you are trying to access the product at BeginRun level. You need to define another token with BeginRun transition and use it here. geomToken_ can be used later in 326 as it is.